01-challenge-problem.py

```diff
@@ -22,14 +22,14 @@ def end_points(self):
         print('End Point Y-Axis (Bottom Left): ' + str(bottom_left))
 
 
-def build_stuff():
+def coordinates():
```

Comment:
Naming functions should be made according to the overall functionality of the function, so a better name would be getCoordinates().